pyfstat/__init__.py

```diff
@@ -33,6 +33,8 @@
 from .injection_parameters import (
     AllSkyInjectionParametersGenerator,
     InjectionParametersGenerator,
+    custom_prior,
+    isotropic_amplitude_priors,
```

Comment:
Shouldn't be at top namespace level, as they're (currently) _not_ the same as used for MCMC sampling.